taxonomies/categories.txt

```diff
@@ -45831,6 +45862,14 @@ ciqual_food_code:en:25164
 ciqual_food_name:en:Osso buco
 ciqual_food_name:fr:Osso buco
 
+<en:Meals with meat
+<en:meat dishes, no garnish
```

Comment:
```suggestion
```

Reply:
Thanks for the info, will be fixed in the next commit